tinygrad/runtime/ops_python.py

```diff
@@ -170,7 +170,8 @@ def a_elem(x, i, j, goff): return x[(i%2)+(j//8)*2+(i//8)*4][goff+((i//2)%4)+(j%
             def b_elem(x, i, j, goff): return x[(j%2)+(j//8)*2][goff+(j//2)%4+(i)*4]
             # (i, j), C, D (4 elements on 32 threads)
             def c_map(lane, elem): return ((elem%2)+(lane%4)*2, (lane//4)+(elem//2)*8)
-            ul[i] = wmma_helper(32, 16, 8, 4, 4, a_elem, b_elem, c_map)
+            sza, szb, szc = (prod(sz for _, sz in upc) for upc in arg[6])
```

Comment:
Instead of arg, you should copy and paste here. I don't like that CLANG uses arg either, this should be a robust test

Reply:
I made it dynamic because they will eventually change based on TC used. For amx, the size changes with the dtype and for cuda it changes according to the mma shape. 

Reverted to fixed values.

Reply:
Sorry, I believe I didn't get the feedback completely. If the arg is replaced with fixed values, I don't see any advantage in calculating the sizes (`sza, szb, szc = (prod(sz for _, sz in upc) ...`) instead of using those directly (completely reverting the changes in ops_python and leaving it as it was.)

Reply:
Reverted changes in ops_python completely